charts/redpanda/helpers.go

```diff
@@ -92,7 +92,7 @@ func StatefulSetPodLabelsSelector(dot *helmette.Dot, statefulSet map[string]any)
 		"app.kubernetes.io/name":      Name(dot),
 	}
 
-	return helmette.Merge(commonLabels, defaults)
+	return helmette.Merge(AdditionalSelectorLabels, defaults)
```

Comment:
```suggestion
	return helmette.Merge(AdditionalSelectorLabels, values.CommonLabels, defaults)
```

That change could be backwards compatible.

Reply:
If we are doing a minor version bump, then let's not.

Reply:
Agree with you